chart/env/dev.yaml

```diff
@@ -172,6 +172,20 @@ workers:
         cpu: 1
         memory: "4Gi"
     tolerations: []
+  -
+    deployName: "light"
+    maxJobsPerNamespace: 1
+    workerOnlyJobTypes: "dataset-split-names-from-streaming,dataset-split-names-from-dataset-info"
+    nodeSelector: {}
+    replicas: 1
+    resources:
+      requests:
+        cpu: 100m
+        memory: "512Mi"
+      limits:
+        cpu: 1
+        memory: "4Gi"
+    tolerations: []
   -
```

Comment:
I would delete all the remaining workers, and increase the number of replicas of generic (renamed to `all`) of the same quantity (in dev/prod/values).

So that we would have only two workers

Reply:
Done in commit https://github.com/huggingface/datasets-server/pull/945/commits/078b4a2b77796683656f829fbfc9ba0c8eaa61e1 only two workers